SampleBrowser/SampleBrowser/UI/Suggestion/SuggestionCellViewModel.swift

```diff
@@ -0,0 +1,44 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+
+struct SuggestionCellViewModel {
+    var title: String
+}
+
+class SuggestionCell: UITableViewCell {
+    static let identifier = "SuggestionIdentifier"
+
+    private lazy var titleLabel: UILabel = .build { _ in }
```

Comment:
Really minor nit: we can just use `.build()` which avoids having to pass in (and run) this empty closure.

```suggestion
    private lazy var titleLabel: UILabel = .build()
```